MANIFEST.in

```diff
@@ -1,2 +0,0 @@
-include LICENSE
-include README.md
```

Comment:
I assuming, removing these.. these files are still included in the distribution file?